### DISCUSSION TITLE
Automatically add screenshots to clipboard

### DISCUSSION THREAD

**Discussion by galbc** [p0deje/Maccy#814] [Q&A] [answered]
Coming from a Windows working environment, I am thinking through the features of the native Windows clipboard manager. 

With that said, one great native feature that Windows offers is that any screenshot taken is automatically copied to the clipboard. If feasible, implementing this would be a great improvement to the functionality of Maccy!

**Comment by druwan** [comment, marked as answer]
In the options for the native screenshot app you can choose to save your screenshots to the clipboard. Then your screenshot shows in Maccy 👍🏽

**Reply by galbc** [reply]
Legend! Thank you

**Comment by hbtmarc** [comment]
I have been using it this way, but I wanted to know if there is a folder accessible through Finder where we can see the screenshot files copied and/or captured using Ctrl+Shift+3, 4, or 5.
Is there? If so, what is the path?
